src/components/EditSections/EditContactInfo/EditContactInfo.js

```diff
@@ -107,6 +125,21 @@ const EditContactInfo = ({
           </Field>
         </Col>
       </Row>
+      <Row>
+        <Col xs={12} md={3}>
+          <Field
+            component={MultiSelection}
+            label={<FormattedMessage id="ui-users.contact.preferredEmailCommunication" />}
+            id="adduser_preferredEmailCommunication"
```

Comment:
Could you please clarify do we need use one notation here (just for example camel case notation)?